artemis-commons/src/main/java/org/apache/activemq/artemis/utils/collections/LinkedListImpl.java

```diff
@@ -175,6 +262,11 @@ public int size() {
       return size;
    }
 
+   /** for tests */
+   public int nodeMapSize() {
```

Comment:
This is an internal class. I don't see a problem on making it public.


I will make a change here though.

Reply:
Notice it is not part of the interface

Reply:
Yep I realised it wasnt part of the interface, but everyone loves to use and depend on public methods that arent on the interface hehe. If its meant for tests I see no reason for it to be higher than minimum visibility.

In this case I'd likely just have used reflection in the test personally as it would probably also better validate the result than just the count does.

Reply:
I'm just renaming the method to something more relevant.. that would be ok if someone went straight to the implementation.